softlayer/vm/softlayer_hardware_creator.go

```diff
@@ -36,7 +37,28 @@ func NewBaremetalCreator(vmFinder Finder, softLayerClient sl.Client, bmsClient b
 }
 
 func (c *baremetalCreator) Create(agentID string, stemcell bslcstem.Stemcell, cloudProps VMCloudProperties, networks Networks, env Environment) (VM, error) {
-	hardwareId, err := c.createBaremetal(cloudProps.VmNamePrefix, cloudProps.BaremetalStemcell, cloudProps.BaremetalNetbootImage)
+	for _, network := range networks {
+		switch network.Type {
+		case "dynamic":
+			if len(network.IP) == 0 {
+				return c.createByBaremetal(agentID, stemcell, cloudProps, networks, env)
+			} else {
+				return c.createByOSReload(agentID, stemcell, cloudProps, networks, env)
+			}
+		case "manual":
+			return nil, bosherr.Error("Support manual netowrk soon...")
```

Comment:
also, change this to `"Manual networking is not currently supported"` let's not make promises in code. Instead use the `develop_v3` branch for that code and when ready we can start having folks alpha test it.